Tests/FluentAssertions.Equivalency.Specs/DataTableSpecs.cs

```diff
@@ -32,6 +32,105 @@ public void When_data_tables_are_both_null_equivalence_test_should_succeed()
         ((DataTable)null).Should().BeEquivalentTo(null);
     }
 
+    [Fact]
+    public void When_row_match_mode_is_invalid_it_should_fail()
+    {
+        // Arrange
+        var typedDataSet1 = CreateDummyDataSet<TypedDataSetSubclass>();
+        var typedDataSet2 = new TypedDataSetSubclass(typedDataSet1);
+
+        var dataTable1 = typedDataSet1.ToUntypedDataSet().Tables["TypedDataTable1"];
+        var dataTable2 = typedDataSet2.ToUntypedDataSet().Tables["TypedDataTable1"];
+
+        // Act
+        Action action = () => dataTable1.Should().BeEquivalentTo(dataTable2, options => options.UsingRowMatchMode((RowMatchMode)2));
+
+        // Assert
+        action.Should().Throw<XunitException>().WithMessage(
+            "Unknown RowMatchMode *when trying to compare *");
+    }
+
+    [Theory]
+    [MemberData(nameof(EmptyPrimaryKeys))]
+    public void When_row_match_mode_is_primary_key_without_primary_key_it_should_fail(DataColumn[] emptyPrimaryKey)
+    {
+        // Arrange
+        var typedDataSet1 = CreateDummyDataSet<TypedDataSetSubclass>(includeRelation: false);
+        var typedDataSet2 = new TypedDataSetSubclass(typedDataSet1);
+
+        var dataTable1 = typedDataSet1.ToUntypedDataSet().Tables["TypedDataTable1"];
+        var dataTable2 = typedDataSet2.ToUntypedDataSet().Tables["TypedDataTable1"];
+
+        dataTable1.PrimaryKey = emptyPrimaryKey;
+
+        // Act
+        Action action = () => dataTable1.Should().BeEquivalentTo(dataTable2, options => options.UsingRowMatchMode(RowMatchMode.PrimaryKey));
```

Comment:
🔧 This line is too long. Please the reformat feature of your IDE so it uses the `.editorconfig` correctly.

Reply:
FYI, VS does not support `max_line_length` https://github.com/dotnet/roslyn/issues/15406
But it seems we can use Roslynator to enforce it https://github.com/JosefPihrt/Roslynator/blob/main/docs/analyzers/RCS0056.md to make it IDE/editor agnostic.